R/selectors.R

```diff
@@ -45,7 +59,7 @@ wrap_to_selector <- function(x, name, parameters = NULL) {
 selector__invert <- function() {
   inverted <- cs$all()$sub(self)
   # TODO: we want to print something like `!cs$all()` when call `!cs$all()`
-  inverted$`_print_override` <- deparse1(sys.call(sys.nframe() - 1))
+  # inverted$`_print_override` <- deparse1(sys.call(sys.nframe() - 1))
```

Comment:
Why?

Reply:
Otherwise, this throws an error:

```r
df$select(!cs$alpha())

$<- - syntax error: Assigning to the
  member `_print_override` of this polars object
  is not supported
```

Reply:
This is to mimic the Python Polars print results as noted in the TODO comment above.
Do you know of a way to achieve this?

Reply:
I tried a couple of things but no success. TBH I don't think the print method of selectors is a priority.

Reply:
Ok, thanks.